lib/core/src/server/serverRequire.js

```diff
@@ -84,7 +84,7 @@ export default function serverRequire(filePath) {
 
   const moduleDescriptor = interpret.extensions[candidateExt];
 
-  if (registerCompiler(moduleDescriptor) === 0) {
+  if (moduleDescriptor !== null && registerCompiler(moduleDescriptor) === 0) {
```

Comment:
So, is the problem only with a `.json`? because there is already a check ☝️  for `.js`.

Reply:
Yeah, you are right. My proposition to not show the warning when `moduleDescriptor` is `undefined` (no key in `interpret.extensions`) or `null` (as for `.js` and `.json`).